### ISSUE TITLE
Refactor a go service to gm-fabric-go

### ISSUE THREAD

**Issue by Minger** [DecipherNow/gm-fabric-go#161]
https://github.com/DecipherNow/team-service/pull/145

> We DEFINITELY have to now spend time updating and refactoring ALL of the go services to the gm-fabricgo SDK...this has to happen by the end of February.

> We should probably pick a service at random and have @alecholmez or @dougfort refactor to use gm-fabric-go. We don't have a clear path for upgrade into fabric so I'd want one of them path finding and laying out the process (and potentially flowing tooling changes back to the team).

> This is a job for @alecholmez because he is already on the MEME contract and he can bill the work. Once we have the process, we can potentially get the other MEME devs engaged like @Minger and @Hoggles to assist.

**Comment by chrismholmes** [comment]
Alec has already done 3 of the go services, so this pattern has been accomplished.  You can certainly ask them to document what they needed to do and follow the pattern.  Yes this must be done prior to end of February.  How many total Go services are there?

**Comment by joshua-rutherford** [comment]
Closing as this is not an issue with this SDK.  This is customer specific work and should be tracked with the appropriate tracking systems for those customers and projects.  

Providing documentation on how to update a service developed for a specific customer using a proprietary SDK that was never published to anyone outside our organization is not in scope of this project.
